apps/web/components/Stepper/Stepper/Stepper.tsx

```
@@ -280,22 +284,55 @@ const Stepper = ({
+            {links && links.length > 0 && (
+              <Stack space={2}>
+                {links
+                  .filter((link) => Boolean(link.href) && Boolean(link.label))
+                  .map((link) => (
+                    <LinkV2
+                      newTab={true}
+                      color="blue400"
+                      underline="normal"
+                      underlineVisibility="always"
+                      href={link.href as string}
+                    >
```

Biome

[error] Missing key property for this element in iterable.

The order of the items may change, and having a key can help React identify which item was moved.
Check the React documentation. 

(lint/correctness/useJsxKeyInIterable)